news/2021-07-26-performance-points-star-rating-updates.md

```diff
@@ -0,0 +1,168 @@
+
+Please note: the current batch of changes contain things that will take effect immediately, as well as ones which will have no visible change for the moment but will lay out the groundwork for improvements in the future.
+
+Additionally, the changes below are split into "star rating" and "performance points" sections, but do note that anything that affects star rating will also impact performance points, as both are based off an algorithmic *skill* value which is dynamically generated from the beatmap being played.
+
+You may also wish to consult the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) articles to broaden your understanding of these surprisingly complicated topics.
+
+As a brief reminder/summary, the following values are a core part of the performance points algorithm:
+
+- **Aim**, which considers how difficult it is to consistently hit notes back-to-back during play (affected by things like approach rate, jumps, Flashlight, Hidden, Hard Rock and more)
+- **Speed**, which considers how quickly new elements are presented to the player during the course of the map (affected by things like the number of hit objects over time, BPM, Double Time and Half Time)
+- **Accuracy**, which considers the player's ability to stay on time rhythmically and how difficult this is to achieve (affected by things like Hidden, Hard Rock, Flashlight and more)
+- **Strain**, which considers how often and how long a player has to endure intense or technically demanding patterns during play (affected by things like streams, fast jumps and more)
+
+## Release Schedule
+
+Deployment of these changes will as usual, take some time to propagate out. As this batch of changes affects all game modes, it is likely to take longer than usual. If you are interested in following the progress we recommend you bookmark this page and check back on this section, as it will be updated as forward progress is made.
+
+### 🏃 Star rating updates \[2021-07-28\]
+
+An initial pass over all beatmaps to update their star ratings. This will take immediate effect for all newly submitted scores.
+
+Estimated to take less than one day.
+
+### 🛑 Background user total recalculations \[2021-07-29\]
+
+Starting from the user at rank #1 and working outwards, each user's total pp will be recalculated.
+
+During this period, rank history graphs will be frozen. If a user plays, their **total PP** value will immediately be updated using the correct values for all past scores, but this won't match past scores' display under "Best Performance". As a result, it may seem as if you temporarily gain, lose or fluctuate in rank relative to others.
```

Comment:
probably fine in lowercase, but feel free to change that

```suggestion
During this period, rank history graphs will be frozen. If a user plays, their **total pp** value will immediately be updated using the correct values for all past scores, but this won't match past scores' display under "Best Performance". As a result, it may seem as if you temporarily gain, lose or fluctuate in rank relative to others.
```